src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/VedtakOgBeregningBarnetilsyn.tsx

```diff
@@ -53,7 +53,7 @@ const VedtakOgBeregningBarnetilsyn: FC = () => {
                                     <AvslåVedtak vedtak={vedtak as AvslagBarnetilsyn} />
                                 )}
                                 {typeVedtak === TypeVedtak.OPPHØR && (
-                                    <OpphørVedtak vedtak={vedtak as OpphørBarnetilsyn} />
+                                    <OpphørVedtak vedtak={vedtak as OpphørBarnetilsynRequest} />
```

Comment:
Denne burde fortsatt være `OpphørBarnetilsyn`, det en den som blir hentet av backend
```suggestion
                                    <OpphørVedtak vedtak={vedtak as OpphørBarnetilsyn} />
```